### PR TITLE
T 2905

### PR DESCRIPTION
**Pull request by maartenkling**
[x] - Does the code comply with the style guide? (Run `make lint` from the Wagtail root)
[x] - For new features: Has the documentation been updated accordingly?

### CODE REVIEW

File: docs/editor_manual/documents_images_snippets/images.rst [hunk 1]
```diff
@@ -26,4 +26,5 @@ ___________
 * This interface allows you to select a focal point which can effect how your image displays to visitors on the front-end.
 * If your images are cropped in some way to make them fit to a specific shape, then the focal point will define the centre point from which the image is cropped.
 * To set the focal point, simply drag a marquee around the most important element of the image.
+* To remove the focal point, simply hit the button below the image.
```
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/images.rst]: Let's remove 'simply' from both these lines.

File: docs/editor_manual/documents_images_snippets/images.rst [hunk 2]
```diff
@@ -26,4 +26,5 @@ ___________
 * This interface allows you to select a focal point which can effect how your image displays to visitors on the front-end.
```
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/images.rst]: Do you think we should replace 'focal point' with 'focal area' throughout?

**Comment by maartenkling** [comment]
@tomdyson updated

File: wagtail/wagtailimages/templates/wagtailimages/images/edit.html [hunk 3]
```diff
@@ -73,7 +73,10 @@ <h2 class="label">{% trans "Focal point (optional)" %}</h2>
                 <img {{ rendition.attrs }} data-original-width="{{ image.width }}" data-original-height="{{ image.height }}" class="show-transparency">
 
                 <div class="current-focal-point-indicator{% if not image.focal_point %} hidden{% endif %}"></div>
+
             </div>
+            <br />
+            <button class="button button-secondary no remove-focal-point">{% trans "Remove focal point" %}</button>
```
Comment (tomdyson) [inline wagtail/wagtailimages/templates/wagtailimages/images/edit.html]: Remove focal point -> Remove focal area

File: docs/editor_manual/documents_images_snippets/images.rst [hunk 4]
```diff
@@ -25,5 +25,6 @@ ___________
 
 * This interface allows you to select a focal point which can effect how your image displays to visitors on the front-end.
 * If your images are cropped in some way to make them fit to a specific shape, then the focal point will define the centre point from which the image is cropped.
```
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/images.rst]: point -> area

File: docs/editor_manual/documents_images_snippets/images.rst [hunk 5]
```diff
@@ -25,5 +25,6 @@ ___________
 
 * This interface allows you to select a focal point which can effect how your image displays to visitors on the front-end.
```
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/images.rst]: point -> area

File: docs/editor_manual/documents_images_snippets/images.rst (hunk 2 above)
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/images.rst]: ...including the heading for this section

**Comment by HvandenBerg** [comment]
I'm unable to set a focal point after having removed the existing focal point. It is possible to edit the existing focal point but not to set a new one after removing it.

**Comment by maartenkling** [comment]
@HvandenBerg re-test please, its fixed. Point is now area.

**Comment by HvandenBerg** [comment]
re-test OK

**Comment by jjanssen** [comment]
Thanks! Merged in 270ac66.

In addition I noticed that the focal point was showing up visually without having it set. Probably due to the enhanced visual border. Turns out there was invalid check within the edit template to determine whether the focal point was set or not. See 4ab57dd

image

**Pull request closed without merging**